terraform/mean-stack-ibmcloud/main.tf

```diff
@@ -0,0 +1,52 @@
+module  "simple_instance" {
+ source = "github.com/jjalvare/simple_vm"
```

Comment:
Should we also push this to the DevOps folder as well?

Reply:
Is added in this PR. I will later make it point to this repo as terraform does not allow me to specify a directory tree other than master.